e2etests/tests/hostconfiguration.go

```diff
@@ -58,6 +57,31 @@ var _ = ginkgo.Describe("Router Host configuration", func() {
 		},
 	}
 
+	vni := v1alpha1.VNI{
```

Comment:
nit: vni / vni1 is a bit harder to follow, can we rename to vni100 / vni200?

Reply:
done